view/template/patient.hbs

```diff
@@ -106,5 +118,67 @@
             allDayDefault: true,
             events: surveys
         });
+
+       
+
 	});
+
+     var scatterChartData = {
+        datasets: [{
+            label: "My First dataset",
+            data: [{
+                    x: 0,
```

Comment:
Then replace the x values with dates.
Like so:
https://github.com/nnnick/Chart.js/blob/v2.0-dev/samples/line-time-point-data.html#L54